Source/EasySynth/Private/CameraRig/YAML/CameraRigYamlInterface.cpp

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2022 YDrive Inc. All rights reserved.
```

Comment:
I don't think anything needed for YAML is needed anymore. If you wish to keep it to support people using OpenCV for calibration that's fair, but it might be worthwhile to unify JSON and YML interfaces a bit more for maintenance reasons

Reply:
Will remove it, just didn't want to delete the code before we were sure we won't use it